src/app/pages/subjects/category-section/book/book.js

```diff
@@ -0,0 +1,52 @@
+import BaseView from '~/helpers/backbone/view';
+import NodeManager from '~/helpers/backbone/node-manager';
+import router from '~/router';
+import {on, props} from '~/helpers/backbone/decorators';
+import {template} from './book.hbs';
+import GetThisTitle from '~/components/get-this-title/get-this-title';
+
+@props({
+    template: template,
+    regions: {
+        getThis: '.get-this-title-container'
+    }
+})
+export default class Book extends BaseView {
+    @on('click img')
+    selectMe() {
+        this.model.set('selectedBook', this.isSelected() ? false : this.data);
+    }
+
+    @on('click .cta > .btn')
+    goToDetails() {
+        router.navigate(`/details?${ this.data.title}`, {
+            trigger: true
+        });
+    }
+
+    constructor(data, model) {
+        super();
+        this.templateHelpers = {
+            coverUrl: data.coverUrl
+        };
+        this.data = data;
+        this.model = model;
+        this.nodeManager = new NodeManager(model);
+        this.nodeManager.manage.css(
+            'selected-book',
+            ['selectedBook'],
+            (value) => ({selected: value === data})
```

Comment:
NodeManager manages DOM nodes. I pre-defined some managers, but the intent is for it to be entirely customizable.
There's a common pattern of program state being reflected in the view: A DOM node's behavior is supposed to be controlled by fetched data (as opposed to static data). We need to keep track of when the node is re-rendered and when the data changes. This comes down to two variables: the node, the data item; plus a bit of code to plug the data item into to get all the data involved, and a boilerplate bit of code to act on that data.
In the HTML, you add a `data-manager=`*nodeName* attribute to the node to tie it to a manager.
In the JS code, you pass the node name, an array of model attribute names, and a function that takes the model attribute values (like require) and returns the final data to be used in the node. (The third argument defaults to the identity function, if you just want the attribute value itself)
To set HTML contents of a node, you'd pass them to `nodeManager.manage.html()`
In onRender, you typically call `nodeManager.updateNodes(this.el)` to refresh its ability to find nodes by name.
The node manager handles updating the node when the value changes, and the one call in onRender handles resetting state in the case of a re-render, but if your template doesn't contain information that varies over the life of the view (and it shouldn't) then you won't need to re-render anyway.